portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,12 +46,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String upperCaseName = request.getParameter("name").toUpperCase();
+        String name = request.getParameter("name");
         String comment = request.getParameter("comment");
```

Comment:
What happens here if the user didn't fill out this form field? If you check that the parameter exists and either fail the request or set a default, you'd be able to ensure that bad data didn't get into your datastore in the first place. (Which is generally good practice, and would greatly simplify your sentiment analysis PR!)